app/models/sql_audit_log.rb

```diff
@@ -0,0 +1,7 @@
+class SqlAuditLog < ApplicationRecord
```

Comment:
should we name this (and the table) `AuditLog` -- since the `Sql` part is kind of implicit?

Reply:
good idea.  will fix.